Cannot instantiate closure

Hi,

I just updated from wasm-bindgen 0.2.15 to 0.2.19, updated Rust nightly as well and now I get this error:

```
error[E0599]: no function or associated item named `new` found for type `wasm_bindgen::prelude::Closure<_>` in the current scope
  --> src/rust\vdom.rs:50:41
   |
50 |         let closure: Closure<FnMut()> = Closure::new();
   |                                         ^^^^^^^^^^^^ function or associated item not found in `wasm_bindgen::prelude::Closure<_>`
```

Has something changed with Closures?

--- a/guide/src/reference/passing-rust-closures-to-js.md
+++ b/guide/src/reference/passing-rust-closures-to-js.md
@@ -63,6 +63,12 @@ return but the JavaScript closure still needs to be valid!
 For this scenario, you need the `Closure` type, which is defined in the
 `wasm_bindgen` crate, exported in `wasm_bindgen::prelude`, and represents a
 "long lived" closure.
+The `Closure` type is currently behind a feature which needs to be enabled:
+
+```toml
+[dependencies]
+wasm-bindgen = {version = "^0.2", features = ["nightly"]}
+```
 
 The validity of the JavaScript closure is tied to the lifetime of the `Closure`
 in Rust. **Once a `Closure` is dropped, it will deallocate its internal memory

--- a/src/closure.rs
+++ b/src/closure.rs
@@ -3,6 +3,8 @@
 //! This module defines the `Closure` type which is used to pass "owned
 //! closures" from Rust to JS. Some more details can be found on the `Closure`
 //! type itself.
+//! 
+//! The `nightly` feature is required for the `Closure` type to be used.
 
 use std::cell::UnsafeCell;
 #[cfg(feature = "nightly")]

